MAUI/DataForm/getting-started.md

```diff
@@ -0,0 +1,491 @@
+* Caption customization: Supports loading the image as caption for the editor.
+* Editors: Built-in support for text, numeric, numeric up-down, picker, date picker, time picker, switch,drop-down,autoComplete and checkbox editors. 
+* Custom editor: Supports loading the custom editors.
+* Validation: Built-in support to validate the data based on the `INotifyDataErrorInfo` and data annotations. It also programmatically supports validation handling.
+
+# Getting Started with Maui DataForm (SfDataForm)
+
+This section explains the quick overview to use the `Maui DataForm (SfDataForm)` for Maui Dataform in your application.
+
+## Creating an application using the .NET MAUI Scheduler
+
+1. Create a new .NET MAUI application in Visual Studio.
+
+2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfDataForm to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.DataForm](https://www.nuget.org/packages/Syncfusion.Maui.DataForm) and then install it.
+
+3. To initialize the control, import the control namespace `Syncfusion.Maui.DataForm` in XAML or C# code.
+
+4. Initialize `SfDataForm.`
+
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="3 5" %}
+
+    <ContentPage   
+            . . .
+            xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+
+            <dataForm:SfDataForm />
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1 9 10" %}
+
+    using Syncfusion.Maui.DataForm;
+    . . .
+
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+    {
+        InitializeComponent();
+        SfDataForm dataForm = new SfDataForm();
+        this.Content = dataForm;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+## Register the handler
+
+The `Syncfusion.Maui.Core` nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the `MauiProgram.cs` file, register the handler for Syncfusion core.
+
+{% tabs %}
+{% highlight C# tabtitle="MauiProgram.cs" hl_lines="1 10" %}
+
+    namespace ValidationModeSample;
+    using Syncfusion.Maui.Core.Hosting;
+
+    public static class MauiProgram
+    {
+	    public static MauiApp CreateMauiApp()
+	    {
+	        var builder = MauiApp.CreateBuilder();
+		    builder
+			    .ConfigureSyncfusionCore()
+			    .UseMauiApp<App>()
+			    .ConfigureFonts(fonts =>
+			    {
+				    fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+				    fonts.AddFont("OpenSans-Semibold.ttf", "OpenSansSemibold");
+			    });
+
+		    return builder.Build();
+	    }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+### Creating data object
+
+The `SfDataForm` is a data edit control so, create a data object to edit the data object.
```

Comment:
create a data object to edit the data object. => change this data object repeated

Reply:
changes done